home/templates/home/prerelease/event_list.html

```diff
@@ -27,10 +27,29 @@ <h3>Events</h3>
             {% endif %}
             </div>
         </div>
+        <h2>Upcoming Events</h2>
         <div class="g-2 g-lg-3 row row-cols-4 justify-content-evenly pt-4">
```

Comment:
```suggestion
        <div class="g-2 g-lg-3 row justify-content-evenly pt-4">
```
This look very small with this property in mobile. If there is already the class `col` in each element in `event_card.html`, it's should be fine normally :)